cl/assets/templates/includes/new_pagination.html

```diff
@@ -0,0 +1,83 @@
+              <a href="?{{ get_string }}page=1"
+                 rel="first" class="btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="hidden-xs hidden-md">First</span>
+              </a>
+            {% endif %}
+            <a href="?{{ get_string }}page={{ page_obj.previous_page_number }}"
+               rel="prev"
+               class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs hidden-md">Prev.</span>
+            </a>
+          {% else %}
+            {% if not hide_first %}
+              <span class="btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="hidden-xs hidden-md">First</span>
+              </span>
+            {% endif %}
+            <span class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs hidden-md">Prev.</span>
+            </span>
+          {% endif %}
+        </div>
+        <div class="col-xs-8 text-center">
+          {% if not hide_numbers %}
+            {% for page in page_obj|page_prev_range %}
+              <a class="btn btn-default" href="?{{ get_string }}page={{ page }}">{{page}}</a>
+            {% endfor %}
+            <span class="btn btn-default disabled"> {{ page_obj.number }}</span>
+            {% for page in page_obj|page_next_range %}
+              <a class="btn btn-default" href="?{{ get_string }}page={{ page }}">{{page}}</a>
+            {% endfor %}
+          {% else %}
+            Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}
+          {% endif %}
+        </div>
+        <div class="col-xs-2 text-right" >
+          {% if page_obj.has_next %}
+            <a href="?{{ get_string }}page={{ page_obj.next_page_number }}"
+               rel="next"
+               class="btn btn-default" >
+			        <span class="hidden-xs hidden-md">Next</span>
+              <i class="fa fa-caret-right no-underline" ></i>
+            </a>
+            {% if not hide_last %}
+              <a href="?{{ get_string }}page={{ page_obj.paginator.num_pages }}"
+                 rel="last" class="btn btn-default" >
+			          <span class="hidden-xs hidden-md">Last</span>
+                <i class="fa fa-caret-right no-underline" ></i>
+                <i class="fa fa-caret-right no-underline" ></i>
+              </a>
+            {% endif %}
+          {% else %}
+            <span class="btn btn-default" >
+			        <span class="hidden-xs hidden-md">Next</span>
+              <i class="fa fa-caret-right no-underline" ></i>
+            </span>
+            {% if not hide_last %}
+              <span class="btn btn-default" >
+			          <span class="hidden-xs hidden-md">Last</span>
```

Comment:
You have `hidden-md` on a bunch of things. How come? Do you have screenshots at each breakpoint? That'd definitely help, if you can provide them.

Reply:
I was attempting to copy what the other template had. 

![image](https://user-images.githubusercontent.com/82225842/232947270-cc3084f8-570e-4664-b22c-b1e9ad108fc2.png)
![image](https://user-images.githubusercontent.com/82225842/232947351-27337d50-a37f-4e38-a31b-32f226674546.png)
![image](https://user-images.githubusercontent.com/82225842/232947469-f91ca7dc-bb1e-48ad-82ee-9cdafde76eb4.png)

I am going to say that the different breakpoints are not right yet.

Reply:
Yeah, I think I'd suggest dropping features as screens get narrower. Start by dropping the words "First," "prev", etc, then drop the page buttons, finally, if you need to, drop the First/Last buttons, but in no case should we wind up on two lines like this.

Reply:
The Bootstrap 3 break points are at 768, 922, and 1200.  Even at the 767 everything still looks good.  It isn't until we get to about 1/2 of the "xs" breakpoint does it start to stack the first/prev and next/last buttons.

We can design for the extra small to be 767, or we can design for extra small meaning some smaller number.  Once I know what the design parameters are, then I can move forward.

Reply:
The iPhone 12/13 mini is 375px and fairly new Samsung phones are 360, so I think it should look good until at least 360.

Reply:
360:
![image](https://user-images.githubusercontent.com/82225842/233206206-f54bb5ca-7d04-4ad3-969c-7774da31a707.png)
767:
![image](https://user-images.githubusercontent.com/82225842/233206302-d96fe9e3-1c68-4b31-a456-d6c19a78256f.png)
768 and 800:
![image](https://user-images.githubusercontent.com/82225842/233205575-8b10082d-6e1c-444f-b7dd-3cd7d46be671.png)
992 and 1199:
![image](https://user-images.githubusercontent.com/82225842/233206567-eeea4bab-068b-4f2c-b83d-f1ad4ff14ce0.png)
1500:
![image](https://user-images.githubusercontent.com/82225842/233206798-703133ce-d395-49f1-a684-7f6ef887dd59.png)
Just some samples.